frontend/src/pages/projects/screens/detail/storage/ClusterStorageTableRow.tsx

```diff
@@ -61,7 +62,7 @@ const ClusterStorageTableRow: React.FC<ClusterStorageTableRowProps> = ({
 
   const selectOptions = availableNotebooks.notebooks.map((notebook) => ({
     value: notebook.metadata.name,
-    content: notebook.metadata.name,
+    content: getDisplayNameFromK8sResource(notebook),
```

Comment:
this is being done here https://github.com/opendatahub-io/odh-dashboard/pull/3584#pullrequestreview-2509549154